components/shop/hero.js

```diff
@@ -0,0 +1,45 @@
+import styles from './hero.module.css'
```

Comment:
Renamed `hero.js` > `Hero.js` for consistency since all components use title case.